doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
+    * Always continue to PR from your own github fork
+      * Branches in the nodejs/node repository are only for release lines
     * [See "Updating Node.js from Upstream"](./onboarding-extras.md#updating-nodejs-from-upstream)
-    * make new branches for all commits you make!
+    * Make new branches for all commits you make!
 
-  * notifications:
-    * use [https://github.com/notifications](https://github.com/notifications) or set up email
-    * watching the main repo will flood your inbox, so be prepared
+  * Notifications:
+    * Use [https://github.com/notifications](https://github.com/notifications) or set up email
+    * Watching the main repo will flood your inbox (several hundred notifications on typical weekdays), so be prepared
 
   * `#node-dev` on [webchat.freenode.net](https://webchat.freenode.net/) is the best place to interact with the CTC / other collaborators
+    * If there are any questions after the session, a good place to ask is there!
+    * Presence is not mandatory, but please drop a note there if force-pushing to `master`
 
 
 ## Project goals & values
 
-  * collaborators are effectively part owners
-    * the project has the goals of its contributors
+  * Collaborators are effectively the collective owners of the project
+    * The project has the goals of its contributors
 
-  * but, there are some higher-level goals and values
-    * not everything belongs in core (if it can be done reasonably in userland, let it stay in userland)
-    * empathy towards users matters (this is in part why we onboard people)
-    * generally: try to be nice to people
+  * There are some higher-level goals and values
+    * Not everything belongs in core (if it can be done reasonably in userland, let it stay in userland)
+    * Empathy towards users matters (this is in part why we onboard people)
+    * Generally: try to be nice to people!
+    * The best outcome is for people who come to our issue tracker to feel like they can come back again.
+
+  * We have a [Code of Conduct][] that you are expected to follow *and* hold others accountable to
 
 ## Managing the issue tracker
 
-  * you have (mostly) free rein – don't hesitate to close an issue if you are confident that it should be closed
-    * **IMPORTANT**: be nice about closing issues, let people know why, and that issues and PRs can be reopened if necessary
-    * Still need to follow the Code of Conduct
+  * You have (mostly) free rein – don't hesitate to close an issue if you are confident that it should be closed
```

Comment:
Micro-nit: Remove the ` - ` and make this two sentences (separated with `.`) or two clauses (separated with `;`).